src/components/Heading/Heading.tsx

```diff
@@ -1,56 +1,82 @@
 import clsx from 'clsx';
-import React, { ReactNode } from 'react';
+import React, { forwardRef } from 'react';
 import styles from './Heading.module.css';
 
-export interface Props {
+export const VARIANTS = [
+  'alert',
+  'base',
+  'brand',
+  'inherit',
+  'neutral',
+  'success',
+  'warning',
+  'white',
   /**
-   * Child node(s) that can be nested inside component
-   */
-  children: ReactNode;
+   * @deprecated Info variant is deprecated.
+   */ 'info',
```

Comment:
I didn't even realize blue was an option for text lol

Reply:
Hmm I had made a ticket to "deprecate the dark tooltip variant" which I used to deprecate in code, but not exactly fully remove. Maybe we can add onto the breaking changes doc?

Reply:
I thought for the dark tooltip variant, we want to announce that we're removing it and then wait a little while so designers have time to adjust. I'm afraid we'll forget in that interim period.

For this blue text, I see it's being used in 3 places, all in the nux popover (`nuxPopover.tsx`, `nuxOnboardingTourBody.tsx`, `nuxOnboardingPopoverBody.tsx`). Those nuxes have blue backgrounds, so we're going to need to decide whether to override the the text to be blue for those (and then phase them out over time) or pick a different color.

Reply:
Ah that makes sense. Then should I add `info` styling (not sure how to pick a color/token here) or since styling can be done via `className`, have the product engineer pass in a styled class?

Reply:
>Then should I add info styling (not sure how to pick a color/token here) or since styling can be done via className, or have the product engineer pass in a styled class?

Sorry, I'm not sure what you mean. Add info styling where/to what?

Reply:
Ah to clarify:
- Currently, there's no styling for `.heading--info` like there are for the other color variants.
- If the `info` variant is being used, but deprecated, does that mean the styling should exist (via `.heading--info`) until we fully announce removal of `info`?
  - If so, which color should I assign to `.heading--info`?

Reply:
Mmm, good question. Since it's only used in 3 places, I would just remove the variant here and override the styles in `traject` since the css variable will still exist there. I don't think we need to keep the variant around because no one's using it anyway. (I'm pretty sure we actually added those 3 headings when we did a heading migration project ourselves.)

Reply:
> For this blue text, I see it's being used in 3 places, all in the nux popover (nuxPopover.tsx, nuxOnboardingTourBody.tsx, nuxOnboardingPopoverBody.tsx). Those nuxes have blue backgrounds, so we're going to need to decide whether to override the the text to be blue for those (and then phase them out over time) or pick a different color.

And some extra context -- on the design side, they want to move the NUXes to be purple. Not sure why they haven't yet though

Reply:
That's really helpful, thanks @anniehu4 !